lib/mongo/database.rb

```diff
@@ -29,6 +29,11 @@ class Database
     # @since 2.0.0
     ADMIN = 'admin'.freeze
 
+    # The external database name.
+    #
+    # @since 2.11.0
+    EXTERNAL = :external
```

Comment:
@p-mongo you mentioned in [RUBY-1563](https://github.com/mongodb/mongo-ruby-driver/pull/1514) preferring inline strings to class constants. I wanted to make this consistent with the ADMIN constant, but I can remove it if you don't want to add more constants.